### PR TITLE
fix(release-health-dropdown): Fix release health dropdown width

### PR DESCRIPTION
**Pull request by priscilawebdev**
closes: https://app.asana.com/0/1182975495223897/1199240177732643/f

### CODE REVIEW

**Comment by matejminar** [comment]
It looks good when the short name is selected:
image
but still looks broken with the long one:
image-2

**Comment by priscilawebdev** [comment]
@matej Thanks. I will double-check it

**Comment by priscilawebdev** [comment]
@matejminar it should be fine now 


image

File: src/sentry/static/sentry/app/components/charts/optionSelector.tsx [hunk 1]
```diff
@@ -19,48 +19,82 @@ type Props = {
...
+    if (menuContainerWidth) {
+      this.setState({menuContainerWidth});
+    }
+  }
+
+  menuContainerRef = React.createRef<HTMLDivElement>();
+
+  render() {
+    const {menuContainerWidth} = this.state;
+    const {options, onChange, selected, title, menuWidth = 'auto'} = this.props;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/charts/optionSelector.tsx:54]: Should this be in default props instead?

**Review by matejminar — APPROVED** [review]
Good job 🚀

File: src/sentry/static/sentry/app/components/charts/optionSelector.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/charts/optionSelector.tsx:54]: sure thing! let me update it.

**Pull request merged**